$NetBSD: patch-ab,v 1.1 2007/11/13 21:19:45 rillig Exp $

<cstdio> does not declare FILE, it only declares std::FILE.

--- src/myspell/hashmgr.hxx.orig	2006-01-06 11:04:02.000000000 +0100
+++ src/myspell/hashmgr.hxx	2007-11-13 21:45:15.000000000 +0100
@@ -1,7 +1,7 @@
 #ifndef _HASHMGR_HXX_
 #define _HASHMGR_HXX_
 
-#include <cstdio>
+#include <stdio.h>
 #include "htypes.hxx"
 
 enum flag { FLAG_CHAR, FLAG_LONG, FLAG_NUM, FLAG_UNI };
